eslogging/handlers.py

```diff
@@ -289,6 +289,17 @@ def _get_es_client(self) -> Elasticsearch:
 
         raise ValueError("Authentication method not supported")
 
+    def create_index_with_mapping(self, index_name: str, mapping: Dict[str, Any]) -> None:
```

Comment:
1. Poti sa o denumesti doar `create_index`, pentru ca mapping este doar un parametru a crearii.

`index_name` - este redundant pentru ca deja exista ca membru in clasa, poti sa-l stergi